app/src/main/java/com/steamclock/steamclogsample/MainActivity.kt

```diff
@@ -106,13 +125,32 @@ class MainActivity : AppCompatActivity() {
                 clog.warn("LogLevel changed to ${clog.config.logLevel.title}")
             }
         }
+
+        // Testing app DataStore to make sure having a Steamclog datastore won't interfere with an
+        // app's local DataStore; this test won't be visible in app, devs can verify datastore read/write
+        // in console logs after the app is launched.
+        lifecycleScope.launch {
+            lifecycle.repeatOnLifecycle(Lifecycle.State.STARTED) {
+                launch {
+                    // Doesn't need to be lifecycle aware for this test
+                    val timestamp =
+                    AppDataStore(baseContext).apply {
```

Comment:
curious to know why `baseContext` is passed in here here, but `applicationContext` is passed into `SClogDataStore`

Reply:
Hrm. Honestly I think that was a discrepancy I missed. Because we're using the context to lookup the datastore instance I don't *think* it matters which we use (although I could be wrong). Since some of the examples I have found seem to make use of the application context, I'll change these cases to explicitly request the `Application` to remove ambiguity.

Reply:
passing in Application is the right call.  Activity Context is tied to the Activity lifecycle, so we don't want any potential issues from carrying out operations tied to an Activity lifecycle.